### ISSUE TITLE
Horizontal Bar Charts is breaking in RTL.

### ISSUE THREAD

**Issue by ravis-farooq** [Abhinandan-Kushwaha/react-native-gifted-charts#337]
I hope this message finds you well. I am reaching out to you as a user and enthusiast of the React Native Gifted Charts library. First of all, I want to express my gratitude for developing such a useful tool for charting in React Native.

However, I have encountered an issue while using the library in right-to-left (RTL) layouts. It appears that the horizontal bar chart component breaks or behaves unexpectedly when displayed in RTL mode. I have thoroughly reviewed the documentation and explored various options, but I haven't been able to find a satisfactory solution.

I understand that implementing RTL support in a library can be challenging due to the complexity of different language and layout orientations. Nonetheless, I wanted to bring this issue to your attention, as I believe it would greatly enhance the usability and versatility of the React Native Gifted Charts library.

I kindly request your guidance and assistance in addressing this RTL issue. If there are any specific instructions, workarounds, or alternative approaches that I may have missed, I would greatly appreciate your advice.

Additionally, if RTL support is not currently available in the library, I humbly request that you consider adding this feature in a future release. It would be tremendously valuable to developers working on projects with RTL language support or RTL-oriented layouts.

Thank you very much for your time and consideration. I look forward to your response and any support you can provide to help resolve this issue. Your contribution to the React Native community is highly appreciated, and I'm confident that with your expertise, we can overcome this challenge.

**Comment by Abhinandan-Kushwaha** [comment]
Hi @ravis-farooq 
Thanks for reminding this.
I will fix this soon and release it in the next version.

**Comment by ravis-farooq** [comment]
@Abhinandan-Kushwaha I am looking for your help regarding this issue.

**Comment by ravis-farooq** [comment]
I am profoundly grateful for your invaluable assistance in crafting this piece. Your guidance has been instrumental in shaping my thoughts and ideas into a coherent and impactful composition. Your expertise and dedication have truly made a remarkable difference, and I couldn't have achieved this without you. Thank you for your time, patience, and commitment to excellence. Your contribution has enriched not only the content but also my learning experience.

**Comment by ravis-farooq** [comment]
@Abhinandan-Kushwaha What is the update?

**Comment by Abhinandan-Kushwaha** [comment]
@ravis-farooq This release will have many new features and many bug fixes along with code refactored. It's going to take some time, maybe  1 more week. Thanks for your patience.

**Comment by ravis-farooq** [comment]
Than

> @ravis-farooq This release will have many new features and many bug fixes along with code refactored. It's going to take some time, maybe 1 more week. Thanks for your patience.

Thanks I will be waiting as i need to integrate it in my project.

**Comment by ravis-farooq** [comment]
This is a gentle reminder about the latest changes that were promised to be pushed in coming days.

**Comment by Abhinandan-Kushwaha** [comment]
Hi @ravis-farooq 
Can you please share your exact requirement, along with screenshots of the expected chart and the buggy chart that you are able to render currently. Also please share your code.

This will help me consider all the edge cases. Also, FYI, all the refactor work on the library has been done and the new version will be released by this weekend

**Comment by ravis-farooq** [comment]
```
import { BarChart } from "react-native-gifted-charts";
        
        const App = () => {
            const barData = [
                {value: 250, label: 'M'},
                {value: 500, label: 'T', frontColor: '#177AD5'},
                {value: 745, label: 'W', frontColor: '#177AD5'},
                {value: 320, label: 'T'},
                {value: 600, label: 'F', frontColor: '#177AD5'},
                {value: 256, label: 'S'},
                {value: 300, label: 'S'},
            ];
            return (
                <View>
                    <BarChart
                        horizontal
                        barWidth={22}
                        noOfSections={3}
                        barBorderRadius={4}
                        frontColor="lightgray"
                        data={barData}
                        yAxisThickness={0}
                        xAxisThickness={0}
                    />
                </View>
            );
        };
```
        
 While using the library in right-to-left (RTL) layouts. It appears that the horizontal bar chart component breaks or behaves unexpectedly when displayed in **RTL mode**.

**Comment by Abhinandan-Kushwaha** [comment]
Hi @ravis-farooq Can you tell how are you implementing the right-to-left (RTL) layouts? We can connect over Zoom or Meet when you are available.

**Comment by ravis-farooq** [comment]
Thanks @Abhinandan-Kushwaha Sure, we can connect through google meet if possible.

**Comment by Abhinandan-Kushwaha** [comment]
Please share your email ID, or drop a mail at kushabhi5@gmail.com

**Comment by ravis-farooq** [comment]
ravisfarooq@gmail.com

**Comment by Abhinandan-Kushwaha** [comment]
@ravis-farooq I have released the new version `1.3.4` Please test it then we can connect.

**Comment by Abhinandan-Kushwaha** [comment]
Please update here after trying the `rtl` prop.
